src/index.js

```diff
@@ -78,7 +78,90 @@ async function listPackages (ghToken, repoName, repoOwner, packageName, packageC
     }
   }
   )
-  return dockerPkg.repository.packages.edges[0].node.versions.edges
+
+  if (dockerPkg.repository.packages.edges[0].node.versions.edges === undefined) {
```

Comment:
we may just want to exit 0 (all well) and log that there are no containers to clean because there are none. I think this runs up against the case where someone includes this action but they have no containers in their repo.

Reply:
along the same lines we might want a scenario where we check if the repository exists first...

Reply:
@ryanjfrizzell I've added the exit and the log for the error! To check if a repo exists via a graphql query I might need help on. I've never worked with graphql before

Reply:
Actually I might have an idea of how to do it...